### DISCUSSION TITLE
Azure Static Web App & Function App Adapter

### DISCUSSION THREAD

**Discussion by JeffEmery** [remix-run/remix#3267] [Proposals]
I have a project that is close to becoming an adapter for Azure Static Web & Function Apps. Is this something the community would want to support? It still needs tests and then probably a template to be more easily implemented. Since it's not going to be part of the Remix repo, I'm not sure it's possible to get it implemented as an option for `create-remix@latest` "Where do you want to deploy?".

Anyway if you want to look at the project it's here https://github.com/JeffEmery/remix-server-azure 

If you think there's value to the community I'm happy to collaborate and do the work to get it production ready.

**Comment by AndrewOttavianoAbb** [comment]
I would really like this! My company is working in Azure and Remix kinda-ish works in Azure, but not super well. It would be great if Azure were added as a first class citizen in the Remix ecosystem. Could this be integrated into the Remix dev server that is currently being developed?

**Comment by arunmmanoharan** [comment]
@JeffEmery Is there a possibility you can update the app to v4 functions?
